Add media: add more context (#44974)
home-assistant/home-assistant.github.io: source/more-info/local-media/add-media.markdown

---
title: "Adding your media to Home Assistant"
description: "More information on how to add your media to Home Assistant."
related:
  - docs: /integrations/media_player/#action-play-media
    title: Play media action
  - docs: /integrations/media_source/#identifying-a-media-source-from-the-media-browser
    title: Identifying a media source from the media browser
---

Home Assistant includes a built-in media browser that lets you browse and play your local media files. This feature is available in all Home Assistant installation types.

On {% term "Home Assistant Operating System" %}, the media folder is automatically created and ready to use. On {% term "Home Assistant Container" %}, you need to mount a volume to `/media` when starting your container.

> ## Accessing the media browser

Reviewer: I would suggest this title to make it clear and consistent with title of last section:
```suggestion
## Accessing the media browser on Home Assistant OS
```

> ## Accessing the media browser

Author: ```suggestion
## Accessing the media browser

The media browser is available on both {% term "Home Assistant Operating System" %} and {% term "Home Assistant Container" %} installations.
```

> ## Accessing the media browser on Home Assistant OS

Author: ```suggestion
## Accessing the media browser
```
the browser works both on os and container.

1. In the sidebar (left side of your Home Assistant interface), go to {% my media_browser title="**Media** > **My media**" %}.
2. From here, you can browse all media files in your `/media` folder and any custom media directories you've configured.

> ## Adding media via the built-in media browser

Reviewer: ```suggestion
## Adding media using the media browser on Home Assistant OS
```

You can add media files using the built-in media browser. This method is ideal for uploading a few files at a time. This method does not let you create folders.

1. Go to {% my media_browser title="**Media** > **My media**" %}.
2. In the top-right corner, select {% icon "mdi:cog" %} **Manage**.
3. Select **Upload** and choose files from your device.
4. If you uploaded the wrong file, select the file and choose **Delete**.

## Removing media via the built-in media browser

You can remove media files using the built-in media browser. This method is ideal for deleting a few files at a time. This method does not let you delete folders.

1. Go to {% my media_browser title="**Media** > **My media**" %}.
2. In the top-right corner, select {% icon "mdi:cog" %} **Manage**.
3. Select one or more files and choose **Delete**.

> ## Adding media using network file sharing (Home Assistant OS)

Reviewer: ```suggestion
## Adding media using network file sharing on Home Assistant OS
```

On {% term "Home Assistant Operating System" %}, the media folder is automatically created and ready to use. On {% term "Home Assistant Container" %}, you need to mount a volume to `/media` when starting your container.

## Accessing the media browser

The media browser is available on both {% term "Home Assistant Operating System" %} and {% term "Home Assistant Container" %} installations.

1. In the sidebar (left side of your Home Assistant interface), go to {% my media_browser title="**Media** > **My media**" %}.
2. From here, you can browse all media files in your `/media` folder and any custom media directories you've configured.

## Adding media using the media browser on Home Assistant OS

You can add media files using the built-in media browser. This method is ideal for uploading a few files at a time. This method does not let you create folders.

1. Go to {% my media_browser title="**Media** > **My media**" %}.
2. In the top-right corner, select {% icon "mdi:cog" %} **Manage**.
3. Select **Upload** and choose files from your device.
4. If you uploaded the wrong file, select the file and choose **Delete**.

> ## Removing media via the built-in media browser

Reviewer: Last one for consistency:
```suggestion
## Removing media using the media browser
```
or if that is the case:
```suggestion
## Removing media using the media browser on Home Assistant OS
```
